README.md

```diff
@@ -96,8 +96,7 @@ r.word_of_the_day()
 
 ## Development
 
- Assuming that you have `[Python](https://www.python.org/downloads/release/python-370/)` and `[pipenv](https://docs.pipenv.org)` installed, set up your environment and install the required dependencies like this instead of the pip `install random-word` defined above:
-
+ Assuming that you have `Python`(https://www.python.org/downloads/release/python-370/ and `pipenv`(https://docs.pipenv.org) installed, set up your environment and install the required dependencies like this instead of the `pip install random-word` defined above:
```

Comment:
@igagankalra I haven't asked you to remove the square bracket. Please see the example below:
```
[`Python`](https://www.python.org/downloads/release/python-370/)
```

Reply:
Yes, I'm doing that.
Thanks. It was by mistake.
And by removing Indentation it looks messy, see that and tell me should I revert indentation back or not? 
@vaibhavsingh97